inference-engine/src/legacy_api/src/transformations/convert_opset1_to_legacy/convert_mul_or_add_finally.cpp

```diff
@@ -176,7 +176,7 @@ ngraph::matcher_pass_callback get_callback() {
 
         bool is_dequantization = (lin_op->get_rt_info().count("DEQUANTIZATION") != 0) && checkElementwise(lin_op);
 
-        if (!is_dequantization && (res == CONVERSION_RESULT::NONE || (res == CONVERSION_RESULT::SCALE_SHIFT && output_shape_rank < 4))) {
+        if (res == CONVERSION_RESULT::NONE || (res == CONVERSION_RESULT::SCALE_SHIFT && output_shape_rank < 4)) {
```

Comment:
We cannot remove this check, because it is affects many cases.
`bool is_dequantization` means that current operation handled in Low Precision transformations and **has per-channel (or scalar-like) constant**.
In case, when `data_shape = [1, 128, 786]` and `constant_shape = [768]`, is_dequantization must be false (non per-channel operation).
I think, we need to update `checkElementwise(lin_op)`. Could you please add check on per-channel constant in this lambda?
The check should be something like this: if `constShape.size() == 1` and `constShape[0] != data_shape[1]`, then return false.

Reply:
You're right, thank you! Let's prepare Func tests to cover this change.

Reply:
I'm sorry, I forgot to clarify one more point. If we have constShape[0] == 1, it is correct case too. In this case we mustn't return false.
I think, existing tests shouldn't fail after this fix.